atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -218,9 +216,73 @@
-                    "text_id": uuid,
-                    "text": text ,
-                    "competency_ids": [competency_to_match["properties"]["competency_id"]]
+            "text_id": uuid,
+            "text": text,
+            "competency_ids": [competency_to_match["properties"]["competency_id"]]
         }
         self.weaviate_client.add_embeddings(CollectionNames.TEXT.value, embedding, properties)
-        return competency_to_match["properties"]["competency_id"]
+        return competency_to_match["properties"]["competency_id"]
+
+    def feedbackLoopPipeline(self, text_id: str, competency_id: str):
```

Ruff (0.11.9)

Function name `feedbackLoopPipeline` should be lowercase

(N802)